static/styles/lib/alerts.scss

```diff
@@ -163,12 +171,14 @@
 			}
 
 			@include media-breakpoint-down(sm) {
-				left: -290px;
+				left: -324px;
```

Comment:
I agree, can you find a different solution without negative values? I remember we had that talk in the past, but don't remember the outcome